prusti/src/bin/prusti-rustc.rs

```diff
@@ -64,8 +64,13 @@ fn process(mut args: Vec<String>) -> Result<(), i32> {
         let prusti_contracts_path =
             get_latest_crate_artefact(&prusti_home, "prusti_contracts", "rlib");
         cmd.arg(format!("prusti_contracts={}", prusti_contracts_path));
+        let dylib_extension = if cfg!(target_os = "macos") {
+            "dylib"
```

Comment:
Isn't there a function in the standard library that returns the correct extension?

Reply:
Unfortunately, I could not find any.